docs/_sidebar.md

```diff
@@ -172,6 +174,7 @@
   - scripts
     - [README](../scripts/README.md)
   - [DOCS_README](DOCS_README.md)
+  - [Theme](theme.md)
```

Comment:
is this also auto generated?

Reply:
Yes. It read through all the md file and updated the doc here.